src/main/java/org/folio/inventory/support/EndpointFailureHandler.java

```diff
@@ -41,7 +45,20 @@ public static void handleFailure(Throwable failure, RoutingContext context) {
     }
   }
 
+  public static <T> Function<Throwable, T> doExceptionally(
+    RoutingContext context) {
+
+    return failure -> {
+      handleFailure(failure, context);
+      return null;
+    };
+  }
+
   public static Throwable getKnownException(Throwable throwable) {
+    if (throwable instanceof AbstractInventoryException) {
+      return throwable;
+    }
+
     return throwable.getCause() instanceof AbstractInventoryException
```

Comment:
Can you use this code that is more readable? This avoid a merge conflict.
https://github.com/folio-org/mod-inventory/pull/185/files#diff-6e5db33f5ac7695ffb682193802ab08cR53-R57

Reply:
Sure, changed.